src/cic/projection.clj

```diff
@@ -1,5 +1,6 @@
 (ns cic.projection
-  (:require [clojure.test.check.random :as r]
+  (:require [cic.model :as model]
+            [clojure.test.check.random :as r]
```

Comment:
some day that random stuff needs to be refactored into its own library. I do see that it is quite big to just copypasta tho